larpandoracontent/LArObjects/LArThreeDSlidingConeFitResult.cc

```diff
@@ -131,6 +132,8 @@ void ThreeDSlidingConeFitResult::GetSimpleConeList(
     TrackStateLinkedList trackStateList;
     CartesianVector directionSum(0.f, 0.f, 0.f);
     const float clusterLength((trackStateMap.begin()->second.GetPosition() - trackStateMap.rbegin()->second.GetPosition()).GetMagnitude());
+    const float lengthStep{0.5f * clusterLength / (nCones - 1)};
+    const float angleStep{0.5f * tanHalfAngle / (nCones - 1)};
```

Comment:
Is there any defence against nCones==1 in the calling function?

Reply:
Actually no... the default value in the main algorithm is 5, but this can be overridden in XML, so I should add a check there